src/index.html

```diff
@@ -21,7 +21,360 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li>
+            <a
+              href="#apple"
+              class="nav__link nav__link--is--active"
+            >
+              apple
+            </a>
```

Comment:
The class name 'nav__link--is--active' is not following the BEM naming convention. It should be 'nav__link--is-active'.